cmd/modules/analyzers/pkg-analyzer/main.go

```diff
@@ -75,9 +75,15 @@ func (pkganalyzer *PkgAnalyzer) Analyze(controlService service.ControlServiceCli
 			}
 
 			for _, target := range pkganalyzer.targetsSlice {
-				re := regexp.MustCompile(filepath.Join(pkganalyzer.targetsDir, target))
-				if re.MatchString(fileNode.Path) {
-					pkgNode.Targets = append(pkgNode.Targets, fileNode)
+				joinedPath := filepath.Join(pkganalyzer.targetsDir, target)
+				if fileNode.Path == joinedPath {
```

Comment:
Why did you remove the regexp? If the targets contain regular expressions this will fail.

Reply:
Right! Will revert this change.

Reply:
Done. Nice catch!